Health/src/main/java/com/sciencesquad/health/health/Nutrition/RealmNutritionModule.java

```diff
@@ -0,0 +1,172 @@
+package com.sciencesquad.health.health.Nutrition;
+
+import io.realm.Realm;
+import io.realm.RealmConfiguration;
+import io.realm.RealmList;
+import io.realm.RealmQuery;
+import io.realm.annotations.RealmModule;
+
+import com.sciencesquad.health.BaseApplication;
+import com.sciencesquad.health.health.database.*;
+import com.sciencesquad.health.health.database.events.*;
+
+
+/**
+ * Created by danielmiller on 2/11/16.
+ *
+ * Realm module for the Nutrition module.
+ * This will be the first example of Realm Integration.
+ */
+
+@RealmModule(classes = {RealmNutritionModel.class})
+public class RealmNutritionModule extends RealmDataContext{
```

Comment:
So does every Module need to replicate this class for their data access? It seems really tedious...